_test/preprocess/manifest.json

```diff
@@ -0,0 +1,85 @@
+{
+    "name": "new feature pre_process",
+    "tests": [
+        "@../../../fylr/test/api/_setup/root_session.json",
+        {
+            "name": "simple preprocess: pipe response through cat, the response must be unchanged",
+            "request": {
+                "endpoint": "settings",
+                "method": "GET"
```

Comment:
Wir können hier leider keine Dependency zu FYLR drin haben. Das braucht eine andere Lösung. Vielleicht ein eigener Server den wir mit in apitest bauen... apitest macht ja eh schon einen webserver auf, das kann man vielleicht nutzen. Mach bitte mal einen Vorschlag.

Reply:
Ich habe den Ordner von `_test` in `apitests` umbenannt und die nötigen Dateien aus dem fylr Repo kopiert. es gibt jetzt keine direkten Abhängigkeiten mehr